frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -59,7 +59,8 @@ final case class Project(
   }
 
   override def toString: String = s"$name"
-  override val hashCode: Int = origin.hash
+  val uniqueId = s"${origin.path}#${name}"
+  override val hashCode: Int = uniqueId.hashCode
```

Comment:
Addressed in the latest commit, the likelihood of this happening is quite small for us because at max we'll only have 200 projects in one bloop server, but it's nonetheless good advice and it's better be safe than sorry!